src/auth_components/UpdateProfile.tsx

```diff
@@ -84,16 +84,18 @@ export default function UpdateProfile() {
       .then(() => {
         // Auth is the source of truth for name/email/password
         // Firestore stores name/email as well but it's still a
-        // Success for the user if Firestore update fails (unlikely)
+        // Success for the user if Firestore update fails
         setMessage("Success! Account updated.");
         console.log("Account updated in Auth.");
 
         Promise.all(firestorePromises)
           .then(() => {
             console.log("Account updated in Firestore.");
           })
-          .catch(() => {
-            console.log("Error updating account in Firestore.");
+          .catch((error) => {
+            console.log(
```

Comment:
I suppose you could use

```suggestion
            console.error(
```